packages/core/components/AutoFrame/index.tsx

```diff
@@ -205,6 +215,14 @@ const CopyHostStyles = ({
     const hrefs: string[] = [];
     let stylesLoaded = 0;
 
+    // Sync attributes for the HTML tag
+    const parentHtml = parentDocument.getElementsByTagName("html")[0];
+    syncAttributes(parentHtml, doc.documentElement);
+
+    // Sync attributes for the Body tag
+    const parentBody = parentDocument.getElementsByTagName("body")[0];
+    syncAttributes(parentBody, doc.body);
```

Comment:
Worth noting that these attributes might change, but this feels like an edge-case that we shouldn't cover just yet.